package.json

```diff
@@ -30,8 +30,12 @@
     "pg": "^7.11.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-modal": "^3.8.2",
```

Comment:
Can we remove the packages that are not used?  this will limit the unnecessary installs pf packages not related to the tool.

Reply:
done